packages/web-components/src/button/button.ts

```diff
@@ -258,9 +258,13 @@ export class BaseButton extends FASTElement {
     return true;
   }
 
-  connectedCallback(): void {
+  public connectedCallback(): void {
     super.connectedCallback();
     this.elementInternals.ariaDisabled = `${!!this.disabledFocusable}`;
+
+    if (this.autofocus) {
```

Comment:
Doesn't having this in the connectedCallback cause it to always steal focus any time it's added to the DOM?